### PR TITLE
[IMP]10730 add module sale_analysis_report.

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by elicoidal** [comment]
Can you check with @xie8899 what is the added value compared with standard Sales order ?

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn @duanyp1991 @seb-elico

File: sale_analysis_report/sale_report.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+
+class sale_report(osv.osv):
+    _inherit = "sale.report"
+    _description = "Sales Orders Statistics"
+    _auto = False
+    _rec_name = 'date'
+
+    _columns = {
+        # TDE FIXME master: rename into date_order
```
Comment (elicoidal) [inline sale_analysis_report/sale_report.py:16]: To be fixed.

File: sale_analysis_report/sale_report.py [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+            'Commitment Delay',
+            digits=(16, 2),
+            readonly=True,
+        ),
+        'categ_id': fields.many2one(
+            'product.category',
+            'Category of Product',
+            readonly=True,
+        ),
+        # TDE FIXME master: rename into nbr_lines
```
Comment (elicoidal) [inline sale_analysis_report/sale_report.py:45]: to be fixed

File: sale_analysis_report/README.rst [hunk 3]
```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Sale Analysis Report
+====================
+
+ This module extends the functionality of Project to add a category on the task and support filter and group.
```
Comment (elicoidal) [inline sale_analysis_report/README.rst:9]: looks incorrect.
Anyway, I am not sure to understand the logic of this module as it seems a simple copy of standard code.

**Pull request closed without merging**